### ISSUE TITLE
Lipics theorem formatting

### ISSUE THREAD

**Issue by dmestel** [PierreSenellart/apxproof#28]
The example produced for lipics-v2016 does not have the lipics formatting for statements of theorems, lemmas, etc.  (For instance `Theorem n' should be boldface, and the statements of theorems should be italicised).

I assume this is because the \newtheoremrep command overrides the theorem environment definitions from the style file.  I get the same behaviour in my own documents with \newtheoremrep and lipics-v2019.

**Comment by PierreSenellart** [comment]
Thanks for the bug report. The problem is actually not with apxproof, but with the LIPIcs stylesheets: these stylesheets use amsthm's `\theoremstyle` to have different styles for theorems, definitions, remarks, and remarks are defined last:
```
\theoremstyle{remark}
\newtheorem*{remark}{Remark}
```
This means that any theorems subsequently defined (such as those defined using `\newtheoremrep`) will have the `remark` style instead of the `plain` style. This problem is apparent in a document not using apxproof, if one were to define, e.g., a `proposition` environment using 
```
\newtheorem{proposition}[theorem]{Proposition}
```

The fix is simple (with or without apxproof): simply add `\theoremstyle{plain}` before the definition of theorems.

You are correct that the provided example did not do that, which resulted in poor formatting of theorems. I have fixed this in 7e14c01. But no fix is needed in apxproof itself.

**Comment by dmestel** [comment]
Thanks!  

By the way, in a similar vein there's a small issue with the interaction with llncs as well.  You can see this if you modify the second line of llncs/test.tex to \usepackage[appendix=inline]{../../apxproof}.  Then theorems defined with \theoremrep are formatted with a space between the number and the period, so as Theorem 1 . rather than Theorem 1.  This only seems to happen in inline mode.

**Comment by PierreSenellart** [comment]
Thanks, this one was an actual bug. Fixed in 53c0468. I also fixed a few other formatting issues. The fixes are for now in the latest development version of the package, https://raw.githubusercontent.com/PierreSenellart/apxproof/develop/apxproof.sty They will be integrated in version 1.1.2 when it is released.
